### PR TITLE
NR-149661-Apache Traffic Server new quickstart.

### PR DESCRIPTION
**Pull request by pkudikyala**
**Jira Ticket:** https://issues.newrelic.com/browse/NR-149661
**Description:** This PR has the new quickstart integration named "Apache Traffic Server"
**Dashboard:** https://onenr.io/02R5bgkrojb

# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: alert-policies/apache-traffic-server/FreeCacheRAM.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Name of the alert
+name: Free Cache RAM Percent
+
+# Description and details
+description: |+
+  This alert is triggered if the cache RAM free lower than 5% for 5 minutes.
```
Comment (mdumpati) [inline alert-policies/apache-traffic-server/FreeCacheRAM.yml:6]:
```suggestion
  This alert is triggered if the free space in RAM memory cache is below 10% for 5 minutes.
```

File: alert-policies/apache-traffic-server/FreeCacheRAM.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+# Function used to aggregate the NRQL query value(s) for comparison to the terms.threshold (Default: SINGLE_VALUE)
+valueFunction: SINGLE_VALUE
+
+# List of Critical and Warning thresholds for the condition
+terms:
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: BELOW
+    # Value that triggers a violation
+    threshold: 5
```
Comment (mdumpati) [inline alert-policies/apache-traffic-server/FreeCacheRAM.yml:24]:
```suggestion
    threshold: 10
```

File: alert-policies/apache-traffic-server/FreeCacheRAM.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    # Time in seconds; 120 - 3600
+    thresholdDuration: 300
+    # How many data points must be in violation for the duration
+    thresholdOccurrences: ALL
+  - priority: WARNING
+    # Operator used to compare against the threshold.
+    operator: BELOW
+    # Value that triggers a violation
+    threshold: 10
```
Comment (mdumpati) [inline alert-policies/apache-traffic-server/FreeCacheRAM.yml:33]:
```suggestion
    threshold: 15
```

File: alert-policies/apache-traffic-server/FreeCacheRAM.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+# Name of the alert
+name: Free Cache RAM Percent
```
Comment (mdumpati) [inline alert-policies/apache-traffic-server/FreeCacheRAM.yml:2]:
```suggestion
name: Free RAM Cache Percent
```

File: alert-policies/apache-traffic-server/SSLExpiredCertificate.yml [hunk 5]
```diff
@@ -0,0 +1,30 @@
+# Name of the alert
+name: SSL Expired Certificates
+
+# Description and details
+description: |+
+  This alert is triggered if the SSL expired certificates exceeds 1 for 5 minutes.
```
Comment (mdumpati) [inline alert-policies/apache-traffic-server/SSLExpiredCertificate.yml:6]:
```suggestion
  This alert is triggered when the SSL certificate is within 15 days of expiration.
```

File: alert-policies/apache-traffic-server/SSLExpiredCertificate.yml [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+# Function used to aggregate the NRQL query value(s) for comparison to the terms.threshold (Default: SINGLE_VALUE)
+valueFunction: SINGLE_VALUE
+
+# List of Critical and Warning thresholds for the condition
+terms:
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: ABOVE
+    # Value that triggers a violation
+    threshold: 1
```
Comment (mdumpati) [inline alert-policies/apache-traffic-server/SSLExpiredCertificate.yml:24]:
```suggestion
    threshold: 7
```

@pkudikyala add warning threshold also

File: dashboards/apache-traffic-server/apache-traffic-server.json [hunk 7]
```diff
@@ -0,0 +1,1583 @@
...
+						"thresholds": [
+							{
+								"alertSeverity": "CRITICAL",
+								"value": 1
+							}
+						]
+					}
+				},
+				{
+					"title": "ATS Free Cache RAM (%)",
```
Comment (mdumpati) [inline dashboards/apache-traffic-server/apache-traffic-server.json:92]:
```suggestion
					"title": "ATS Free RAM Cache (%)",
```

File: dashboards/apache-traffic-server/apache-traffic-server.json [hunk 8]
```diff
@@ -0,0 +1,1583 @@
...
+						"id": "viz.billboard"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"nrqlQueries": [
+							{
+								"accountIds": [],
+								"query": "SELECT (latest(global.proxy.process.cache.ram_cache.total_bytes) - latest(global.proxy.process.cache.ram_cache.bytes_used)) / latest(global.proxy.process.cache.ram_cache.total_bytes)*100  AS '(%) free cache RAM' FROM ATSCacheSampleMetrics"
```
Comment (mdumpati) [inline dashboards/apache-traffic-server/apache-traffic-server.json:110]:
```suggestion
								"query": "SELECT (latest(global.proxy.process.cache.ram_cache.total_bytes) - latest(global.proxy.process.cache.ram_cache.bytes_used)) / latest(global.proxy.process.cache.ram_cache.total_bytes)*100  AS '(%) free RAM cache' FROM ATSCacheSampleMetrics"
```

File: dashboards/apache-traffic-server/apache-traffic-server.json [hunk 9]
```diff
@@ -0,0 +1,1583 @@
...
+								"query": "SELECT latest(global.proxy.process.cache_total_requests) AS 'total cache requests' FROM ATSCacheSampleMetrics"
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						}
+					}
+				},
+				{
+					"title": "Total Cache RAM",
```
Comment (mdumpati) [inline dashboards/apache-traffic-server/apache-traffic-server.json:597]:
```suggestion
					"title": "Total RAM Cache",
```

File: dashboards/apache-traffic-server/apache-traffic-server.json [hunk 10]
```diff
@@ -0,0 +1,1583 @@
...
+						"id": "viz.billboard"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"nrqlQueries": [
+							{
+								"accountIds": [],
+								"query": "SELECT latest(global.proxy.process.cache.ram_cache.total_bytes) / 1e+6 AS 'total cache RAM (MB)' FROM ATSCacheSampleMetrics"
```
Comment (mdumpati) [inline dashboards/apache-traffic-server/apache-traffic-server.json:615]:
```suggestion
								"query": "SELECT latest(global.proxy.process.cache.ram_cache.total_bytes) / 1e+6 AS 'total RAM cache (MB)' FROM ATSCacheSampleMetrics"
```

File: dashboards/apache-traffic-server/apache-traffic-server.json [hunk 11]
```diff
@@ -0,0 +1,1583 @@
...
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						}
+					}
+				},
+				{
+					"title": "Free Cache RAM",
```
Comment (mdumpati) [inline dashboards/apache-traffic-server/apache-traffic-server.json:624]:
```suggestion
					"title": "Free RAM Cache",
```

File: dashboards/apache-traffic-server/apache-traffic-server.json [hunk 12]
```diff
@@ -0,0 +1,1583 @@
...
+						"id": "viz.billboard"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"nrqlQueries": [
+							{
+								"accountIds": [],
+								"query": "SELECT (latest(global.proxy.process.cache.ram_cache.total_bytes) - latest(global.proxy.process.cache.ram_cache.bytes_used)) / 1e+6 AS 'free cache RAM (MB)' FROM ATSCacheSampleMetrics"
```
Comment (mdumpati) [inline dashboards/apache-traffic-server/apache-traffic-server.json:642]:
```suggestion
								"query": "SELECT (latest(global.proxy.process.cache.ram_cache.total_bytes) - latest(global.proxy.process.cache.ram_cache.bytes_used)) / 1e+6 AS 'free RAM cache (MB)' FROM ATSCacheSampleMetrics"
```

File: dashboards/apache-traffic-server/apache-traffic-server.json [hunk 13]
```diff
@@ -0,0 +1,1583 @@
...
+								"query": "SELECT latest(global.proxy.process.http.total_incoming_connections) AS 'incoming connections', latest(global.proxy.process.http.incoming_responses) AS 'incoming responses' FROM ATSHttpSampleMetrics TIMESERIES AUTO "
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						}
+					}
+				},
+				{
+					"title": "HTTP Average Transactions Per Client & Server",
```
Comment (mdumpati) [inline dashboards/apache-traffic-server/apache-traffic-server.json:1095]:
```suggestion
					"title": "HTTP Average Transactions per Client & Server",
```

File: dashboards/apache-traffic-server/ats-08.png
Comment (mdumpati) [inline dashboards/apache-traffic-server/ats-08.png:1]: @pkudikyala In the below screenshot variable field not showing can you take the screenshot along with variable field?

File: data-sources/apache-traffic-server/config.yml [hunk 14]
```diff
@@ -0,0 +1,14 @@
...
+icon: logo.png
+install:
+  primary:
+    link:
+      url: https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration/
+keywords:
+  - apache-traffic-server
+  - apache
+  - ats
+  - NR1_addData
```
Comment (mdumpati) [inline data-sources/apache-traffic-server/config.yml:14]:
```suggestion
```
@pkudikyala Can you remove this till the @rahulbasu-nr gap analysis need to complete.

File: quickstarts/apache-traffic-server/config.yml [hunk 15]
```diff
@@ -0,0 +1,52 @@
...
+  - name: Documentation on how to integrate Apache Traffic Server
+    description: |
+      Monitor the performance metrics of your Apache Traffic Server instances in real-time with New Relic Apache Traffic Server.
+    url: >-
+      https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration/
+keywords:
+  - apache-traffic-server
+  - apache
+  - ats
+  - NR1_addData
```
Comment (mdumpati) [inline quickstarts/apache-traffic-server/config.yml:46]:
```suggestion
```

here as well

**Review by mdumpati — CHANGES_REQUESTED** [review]
@pkudikyala Check my comments
Note: From now onwards dashboard title was change to Sentence case (https://docs.newrelic.com/docs/style-guide/ui-writing/capitalization/)

File: alert-policies/apache-traffic-server/FreeCacheRAM.yml (hunk 1 above)
Comment (pkudikyala) [reply alert-policies/apache-traffic-server/FreeCacheRAM.yml]: Hi @mdumpati,
As per your suggestions, I have made changes. Kindly review it.

Thanks.

File: alert-policies/apache-traffic-server/FreeCacheRAM.yml (hunk 2 above)
Comment (pkudikyala) [reply alert-policies/apache-traffic-server/FreeCacheRAM.yml]: Hi @mdumpati,
As per your suggestions, I have made changes. Kindly review it.

Thanks.

File: alert-policies/apache-traffic-server/FreeCacheRAM.yml (hunk 3 above)
Comment (pkudikyala) [reply alert-policies/apache-traffic-server/FreeCacheRAM.yml]: Hi @mdumpati,
As per your suggestions, I have made changes. Kindly review it.

Thanks.

File: alert-policies/apache-traffic-server/FreeCacheRAM.yml (hunk 4 above)
Comment (pkudikyala) [reply alert-policies/apache-traffic-server/FreeCacheRAM.yml]: Hi @mdumpati,
As per your suggestions, I have made changes. Kindly review it.

Thanks.

File: alert-policies/apache-traffic-server/SSLExpiredCertificate.yml (hunk 5 above)
Comment (pkudikyala) [reply alert-policies/apache-traffic-server/SSLExpiredCertificate.yml]: Hi @mdumpati,
As per your suggestions, I have made changes. Kindly review it.

Thanks.

File: dashboards/apache-traffic-server/apache-traffic-server.json (hunk 12 above)
Comment (pkudikyala) [reply dashboards/apache-traffic-server/apache-traffic-server.json]: Hi @mdumpati,
As per your suggestions, I have made changes. Kindly review it.

Thanks.

File: dashboards/apache-traffic-server/ats-08.png
Comment (pkudikyala) [reply dashboards/apache-traffic-server/ats-08.png:1]: Hi @mdumpati,
As per your suggestions, I have added a new screenshot. Kindly review it.

Thanks.

File: data-sources/apache-traffic-server/config.yml (hunk 14 above)
Comment (pkudikyala) [reply data-sources/apache-traffic-server/config.yml]: Hi @mdumpati,
As per your suggestions, I have removed the keyword. Kindly review it.

Thanks.

File: quickstarts/apache-traffic-server/config.yml (hunk 15 above)
Comment (pkudikyala) [reply quickstarts/apache-traffic-server/config.yml]: Hi @mdumpati,
As per your suggestions, I have removed the keyword. Kindly review it.

Thanks.

File: dashboards/apache-traffic-server/apache-traffic-server.json (hunk 13 above)
Comment (pkudikyala) [reply dashboards/apache-traffic-server/apache-traffic-server.json]: Hi @mdumpati,
As per your suggestions, I have made changes. Kindly review it.

Thanks.

File: alert-policies/apache-traffic-server/SSLExpiredCertificate.yml (hunk 6 above)
Comment (pkosanam) [reply alert-policies/apache-traffic-server/SSLExpiredCertificate.yml]: done

File: quickstarts/apache-traffic-server/config.yml [hunk 16]
```diff
@@ -0,0 +1,51 @@
...
+  - Early issue detection: Get alerts when there are potential problems with your ATS deployment so you can take action quickly.
+  - Capacity planning: Plan for future growth by understanding how your ATS deployment is currently being used.
+  - Efficient troubleshooting: Troubleshoot problems with your ATS deployment quickly and easily by using New Relic's tools.
+  - User experience enhancement: Improve the user experience of your applications by understanding how they are being used and making changes to improve performance.
+  - Security insights: Get insights into the security of your ATS deployment and identify potential threats.
+  - Continuous improvement: Use New Relic's data to continuously improve the performance and security of your ATS deployment.
+  - Business impact: Understand the impact of your ATS deployment on your business by measuring metrics such as revenue, traffic, and customer satisfaction.
+
+  A well-monitored Apache Traffic Server (ATS) can help you improve the performance of your online services, which can lead to increased user engagement, longer site visits, and higher profits.
```
Comment (pkosanam) [inline quickstarts/apache-traffic-server/config.yml:27]:
```suggestion
  Monitoring Apache Traffic Server (ATS) with New Relic can help you improve the performance of your online services, which can lead to increased user engagement, longer site visits, and higher revenue.
```

File: quickstarts/apache-traffic-server/config.yml [hunk 17]
```diff
@@ -0,0 +1,51 @@
...
+documentation:
+  - name: Documentation on how to integrate Apache Traffic Server
+    description: |
+      Monitor the performance metrics of your Apache Traffic Server instances in real-time with New Relic Apache Traffic Server.
+    url: >-
+      https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration/
+keywords:
+  - apache-traffic-server
+  - apache
+  - ats
```
Comment (pkosanam) [inline quickstarts/apache-traffic-server/config.yml:45]:
```suggestion
  - ats
  - apache traffic server
  - web proxy
  - web proxy cache
  - HTTP caching
  - reverse proxy
  - content filtering
```

**Review by pkosanam — CHANGES_REQUESTED** [review]
please change dashboard images as per the changes applied on the dashboard

File: quickstarts/apache-traffic-server/config.yml [hunk 18]
```diff
@@ -0,0 +1,51 @@
...
+  
+  ## Why is it important to monitor Apache Traffic Server?
+  By leveraging New Relic, you can gain a more in-depth understanding of Monitoring Apache Traffic Server is an essential procedure that provides priceless information about the functionality, performance, and general health of this potent web caching and proxying solution. You may obtain a thorough insight of ATS's behaviour, spot any problems before they become serious, and make sure that your web infrastructure is working as efficiently as possible by closely monitoring it. 
+
+  It's a proactive strategy for guaranteeing outstanding web performance, exceptional user experiences, and the dependable functioning of your web infrastructure. You can efficiently manage ATS and keep up a solid, high-performance web infrastructure by New Relic ATS..
```
[the remark this answers is not in the document]
Comment (pkudikyala) [reply quickstarts/apache-traffic-server/config.yml]: Hi @pkosanam ,
As per your suggestion, I have made changes. Kindly review it.

Thanks...

File: quickstarts/apache-traffic-server/config.yml [hunk 19]
```diff
@@ -0,0 +1,51 @@
...
+
+  - Real-time visibility: Get insights into the performance of your ATS deployment in real time.
+  - Optimized performance: Identify performance bottlenecks and optimize your ATS configuration to improve performance.
+  - Early issue detection: Get alerts when there are potential problems with your ATS deployment so you can take action quickly.
+  - Capacity planning: Plan for future growth by understanding how your ATS deployment is currently being used.
+  - Efficient troubleshooting: Troubleshoot problems with your ATS deployment quickly and easily by using New Relic's tools.
+  - User experience enhancement: Improve the user experience of your applications by understanding how they are being used and making changes to improve performance.
+  - Security insights: Get insights into the security of your ATS deployment and identify potential threats.
+  - Continuous improvement: Use New Relic's data to continuously improve the performance and security of your ATS deployment.
+  - Business impact: Understand the impact of your ATS deployment on your business by measuring metrics such as revenue, traffic, and customer satisfaction.
```
[the remark this answers is not in the document]
Comment (pkudikyala) [reply quickstarts/apache-traffic-server/config.yml]: Hi @pkosanam ,
As per your suggestion, I have made changes. Kindly review it.

Thanks...

File: quickstarts/apache-traffic-server/config.yml (hunk 16 above)
Comment (pkudikyala) [reply quickstarts/apache-traffic-server/config.yml]: Hi @pkosanam ,
As per your suggestion, I have made changes. Kindly review it.

Thanks...

File: quickstarts/apache-traffic-server/config.yml (hunk 17 above)
Comment (pkudikyala) [reply quickstarts/apache-traffic-server/config.yml:42]: Hi @pkosanam ,
As per your suggestion, I have made changes. Kindly review it.

Thanks...

File: alert-policies/apache-traffic-server/CPUPercent.yml [hunk 20]
```diff
@@ -0,0 +1,40 @@
+# Name of the alert
+name: CPU Percent
+
+# Description and details
+description: |+
+  This alert is triggered if the CPU usage exceeds 95% for 5 minutes.
```
Comment (pkosanam) [inline alert-policies/apache-traffic-server/CPUPercent.yml:6]:
```suggestion
  This alert is triggered if the CPU usage exceeds 90% for 5 minutes.
```

File: alert-policies/apache-traffic-server/CPUPercent.yml [hunk 21]
```diff
@@ -0,0 +1,40 @@
...
+# Function used to aggregate the NRQL query value(s) for comparison to the terms.threshold (Default: SINGLE_VALUE)
+valueFunction: SINGLE_VALUE
+
+# List of Critical and Warning thresholds for the condition
+terms:
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: ABOVE
+    # Value that triggers a violation
+    threshold: 95
```
Comment (pkosanam) [inline alert-policies/apache-traffic-server/CPUPercent.yml:24]:
```suggestion
    threshold: 90
```

File: alert-policies/apache-traffic-server/CPUPercent.yml [hunk 22]
```diff
@@ -0,0 +1,40 @@
...
+    # Time in seconds; 120 - 3600
+    thresholdDuration: 300
+    # How many data points must be in violation for the duration
+    thresholdOccurrences: ALL
+  - priority: WARNING
+    # Operator used to compare against the threshold.
+    operator: ABOVE
+    # Value that triggers a violation
+    threshold: 90
```
Comment (pkosanam) [inline alert-policies/apache-traffic-server/CPUPercent.yml:33]:
```suggestion
    threshold: 85
```

File: alert-policies/apache-traffic-server/MemoryUsagePercent.yml [hunk 23]
```diff
@@ -0,0 +1,40 @@
+# Name of the alert
+name: Memory Usage Percent
+
+# Description and details
+description: |+
+  This alert is triggered if the memory usage exceeds 95% for 5 minutes.
```
Comment (pkosanam) [inline alert-policies/apache-traffic-server/MemoryUsagePercent.yml:6]:
```suggestion
  This alert is triggered if the memory usage exceeds 90% for 5 minutes.
```

File: alert-policies/apache-traffic-server/MemoryUsagePercent.yml [hunk 24]
```diff
@@ -0,0 +1,40 @@
...
+# Function used to aggregate the NRQL query value(s) for comparison to the terms.threshold (Default: SINGLE_VALUE)
+valueFunction: SINGLE_VALUE
+
+# List of Critical and Warning thresholds for the condition
+terms:
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: ABOVE
+    # Value that triggers a violation
+    threshold: 95
```
Comment (pkosanam) [inline alert-policies/apache-traffic-server/MemoryUsagePercent.yml:24]:
```suggestion
    threshold: 90
```

File: alert-policies/apache-traffic-server/MemoryUsagePercent.yml [hunk 25]
```diff
@@ -0,0 +1,40 @@
...
+    # Time in seconds; 120 - 3600
+    thresholdDuration: 300
+    # How many data points must be in violation for the duration
+    thresholdOccurrences: ALL
+  - priority: WARNING
+    # Operator used to compare against the threshold.
+    operator: ABOVE
+    # Value that triggers a violation
+    threshold: 90
```
Comment (pkosanam) [inline alert-policies/apache-traffic-server/MemoryUsagePercent.yml:33]:
```suggestion
    threshold: 85
```

File: alert-policies/apache-traffic-server/SSLExpiredCertificate.yml [hunk 26]
```diff
@@ -0,0 +1,30 @@
...
+# Function used to aggregate the NRQL query value(s) for comparison to the terms.threshold (Default: SINGLE_VALUE)
+valueFunction: SINGLE_VALUE
+
+# List of Critical and Warning thresholds for the condition
+terms:
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: ABOVE
+    # Value that triggers a violation
+    threshold: 7
```
Comment (pkosanam) [inline alert-policies/apache-traffic-server/SSLExpiredCertificate.yml:24]: should this be changed ?
as the alert has to be within 15 days.

File: dashboards/apache-traffic-server/apache-traffic-server.json [hunk 27]
```diff
@@ -0,0 +1,1583 @@
...
+						"row": 2,
+						"width": 2,
+						"height": 3
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.markdown"
+					},
+					"rawConfiguration": {
+						"text": "\n\n**About**\n\nInstrument your application with New Relic - [Add Data](https://one.newrelic.com/marketplace?account=3510613).\n\nUnable to find data in your dashboard? -[Troubleshoot here](\nhttps://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/flex-integration-tool-build-your-own-integration/)\n\nFollow New Relic [Apache Traffic Server documentation](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration/) to instrument Apache Traffic Server.\n\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=ApacheTrafficServer) here and let us know how we can improve it for you.\n"
```
Comment (pkosanam) [inline dashboards/apache-traffic-server/apache-traffic-server.json:195]:
```suggestion
						"text": "\n\n**About**\n\nInstrument your application with New Relic - [Add Data](https://one.newrelic.com/marketplace?account=3510613).\n\nUnable to find data in your dashboard? -\nInstrument Apache Traffic Server with New Relic using the [documentation](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration/).\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=ApacheTrafficServer) here and let us know how we can improve it for you.\n"
```

File: quickstarts/apache-traffic-server/config.yml [hunk 28]
```diff
@@ -0,0 +1,54 @@
+
+
+slug: apache-traffic-server
+description: |
+  ## Comprehensive monitoring quickstart for Apache Traffic Server
+  By integrating Apache Traffic Server (ATS) with New Relic, you can efficiently set up a monitoring infrastructure that captures, visualizes, and alerts you to important ATS metrics. Monitoring ATS's performance and health ensures smooth operation, rapid issue identification, and proactive problem resolution, ultimately enhancing the reliability and efficiency of your web infrastructure.
```
Comment (pkosanam) [inline quickstarts/apache-traffic-server/config.yml:6]:
```suggestion
  By integrating Apache Traffic Server (ATS) with New Relic, you can efficiently set up a monitoring infrastructure that captures, visualizes, and alerts you to important ATS metrics. Monitoring ATS performance and health ensures smooth operation, rapid issue identification, and proactive problem resolution, ultimately enhancing the reliability and efficiency of your web infrastructure.
```

File: quickstarts/apache-traffic-server/config.yml [hunk 29]
```diff
@@ -0,0 +1,54 @@
...
+  
+  ## Why is it important to monitor Apache Traffic Server?
+  By leveraging New Relic, you can gain a more in-depth understanding of Monitoring Apache Traffic Server is an essential procedure that provides priceless information about the functionality, performance, and general health of this potent web caching and proxying solution. You may obtain a thorough insight of ATS's behaviour, spot any problems before they become serious, and make sure that your web infrastructure is working as efficiently as possible by closely monitoring it.
```
Comment (pkosanam) [inline quickstarts/apache-traffic-server/config.yml:9]:
```suggestion
  By leveraging New Relic, you can gain a more in-depth understanding of Monitoring Apache Traffic Server is an essential procedure that provides priceless information about the functionality, performance, and general health of this potent web caching and proxying solution. You will get a comprehensive understanding of ATS behavior, spot any problems before they become serious, and make sure that your web infrastructure is working as efficiently as possible by closely monitoring it.
```

File: quickstarts/apache-traffic-server/config.yml [hunk 30]
```diff
@@ -0,0 +1,54 @@
...
+  
+  ## Why is it important to monitor Apache Traffic Server?
+  By leveraging New Relic, you can gain a more in-depth understanding of Monitoring Apache Traffic Server is an essential procedure that provides priceless information about the functionality, performance, and general health of this potent web caching and proxying solution. You may obtain a thorough insight of ATS's behaviour, spot any problems before they become serious, and make sure that your web infrastructure is working as efficiently as possible by closely monitoring it. 
+
+  Install this quickstart as part of your proactive observability strategy to guarantee outstanding web performance, exceptional user experiences, and dependable functioning of your web infrastructure. You can efficiently manage ATS and keep up a solid, high-performance web infrastructure.
+
+
+  ## What’s included in this quickstart?
+  The New Relic Apache Traffic Server monitoring quickstart provides out-of-the-box reporting that covers the following areas:
```
Comment (pkosanam) [inline quickstarts/apache-traffic-server/config.yml:15]:
```suggestion
  The Apache Traffic Server quickstart provides out-of-the-box reporting that covers the following areas:
```

**Review by pkosanam — CHANGES_REQUESTED** [review]

File: alert-policies/apache-traffic-server/CPUPercent.yml (hunk 20 above)
Comment (pkudikyala) [reply alert-policies/apache-traffic-server/CPUPercent.yml]: Hi @pkosanam ,
As per your suggestion, I have made changes. Kindly review it.

Thanks...

File: alert-policies/apache-traffic-server/CPUPercent.yml (hunk 21 above)
Comment (pkudikyala) [reply alert-policies/apache-traffic-server/CPUPercent.yml]: Hi @pkosanam ,
As per your suggestion, I have made changes. Kindly review it.

Thanks...

File: dashboards/apache-traffic-server/apache-traffic-server.json (hunk 27 above)
Comment (pkudikyala) [reply dashboards/apache-traffic-server/apache-traffic-server.json]: Hi @pkosanam ,
As per your suggestion, I have made changes. Kindly review it.

Thanks...

File: quickstarts/apache-traffic-server/config.yml (hunk 29 above)
Comment (pkudikyala) [reply quickstarts/apache-traffic-server/config.yml]: Hi @pkosanam ,
As per your suggestion, I have made changes. Kindly review it.

Thanks...

File: quickstarts/apache-traffic-server/config.yml (hunk 30 above)
Comment (pkudikyala) [reply quickstarts/apache-traffic-server/config.yml]: Hi @pkosanam ,
As per your suggestion, I have made changes. Kindly review it.

Thanks...

File: quickstarts/apache-traffic-server/config.yml (hunk 28 above)
Comment (pkudikyala) [reply quickstarts/apache-traffic-server/config.yml]: Hi @pkosanam ,
As per your suggestion, I have made changes. Kindly review it.

Thanks...

File: alert-policies/apache-traffic-server/SSLExpiredCertificate.yml (hunk 26 above)
Comment (pkudikyala) [reply alert-policies/apache-traffic-server/SSLExpiredCertificate.yml]: Hi @pkosanam ,
Here changed the entire alert description and threshold. Kindly review it.

Thanks...

Thanks...

**Review by seemantk — APPROVED** [review]
lgtm

**Pull request merged**